index.bs

```diff
@@ -244,6 +246,10 @@ The {{XRFrame/createAnchor(pose, space)}} method, when invoked on an {{XRFrame}}
 
 Note: It is the responsibility of user agents to ensure that the physical origin tracked by the anchor returned by each {{XRFrame/createAnchor(pose, space)}} call aligns as closely as possible with the physical location of <var ignore=''>pose</var> within <var ignore=''>space</var> at the time represented by the frame on which the method is called. Specifically, this means that for spaces that are dynamically changing, user agents should attempt to capture the native origin of such spaces at the app's specified time. This text is non-normative, but expresses the intent of the specification author(s) and contributors and thus it is highly recommended that it is followed by the implementations to ensure consistent behavior across different vendors.
 
+<div class="algorithm unstable">
+{{XRSession/persistentAnchors}} returns a list of the known persistent anchors. This list MUST reflect the keys of the [=XRSession/map of persistent anchors=].
```

Comment:
Did we already specify how this map is initially populated (on session creation)?